src/styles/StyleUtils.js

```diff
@@ -733,6 +733,54 @@ function getReportWelcomeContainerStyle(isSmallScreenWidth) {
     };
 }
 
+/**
+ * Gets the correct styles for emoji suggestion item
+ *
+ * @param {number} highlightedEmojiIndex
+ * @param {number} rowHeight
+ * @param {Boolean} hovered
+ * @param {number} currentEmojiIndex
+ * @returns {Object}
+ */
+function getEmojiSuggestionItemStyle(
+    highlightedEmojiIndex,
+    rowHeight,
+    hovered,
+    currentEmojiIndex,
+) {
+    return [
+        {
+            height: rowHeight,
+            justifyContent: 'center',
+        },
+        styles.ph3,
+        (currentEmojiIndex === highlightedEmojiIndex && !hovered) || hovered
+            ? {
+                backgroundColor: themeColors.highlightBG,
+            }
+            : {},
+    ];
+}
+
+/**
+ * Gets the correct position for emoji suggestion container
+ *
+ * @param {number} itemsHeight
+ * @param {Boolean} shouldIncludeReportRecipientLocalTimeHeight
+ * @returns {Object}
+ */
+function getEmojiSuggestionContainerStyle(
+    itemsHeight,
+    shouldIncludeReportRecipientLocalTimeHeight,
+) {
+    const reportRecipientLocalTimeHeight = 25;
```

Comment:
We should move this to const

Reply:
Just to clarify, the height value for this component will be derived from the combined padding and margins of a few views(from the RecipientLocalTime component). Therefore, this constant will only be used here. Is that okay with you?